fe/src/main/java/org/apache/doris/load/DeleteHandler.java

```diff
@@ -478,9 +479,11 @@ private void checkDeleteV2(OlapTable table, Partition partition, List<Predicate>
             }
 
             Column column = nameToColumn.get(columnName);
-            if (!column.isKey()) {
+            if (!column.isKey() && table.getKeysType() != KeysType.DUP_KEYS
```

Comment:
should add some comment to explain why float and double type delete predicate can not be supported.